pkgs/tools/misc/pb_cli/default.nix

```diff
@@ -7,17 +7,15 @@ assert clipboard -> xclip != null;
 
 stdenv.mkDerivation rec {
   name = "pb_cli-${version}";
-  version = "1.0";
+  version = "2.0";
```

Comment:
Thanks for the update! Please correct me if I'm wrong, but it looks like this version number 2.0 is something invented for Nixpkgs.

https://nixos.org/nixpkgs/manual/#sec-package-naming

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

Reply:
thanks, fixed